cmd/worklog.go

```diff
@@ -175,6 +176,19 @@ func worklogResolve(ctx *cli.Context) error {
 
 	w := tabwriter.NewWriter(os.Stdout, 2, 0, 2, ' ', 0)
 	for _, item := range toResolve {
+		if item.Type() == apitypes.InviteApproveWorklogType {
+			w.Flush()
+
+			err = AskPerform("Approve invite for " + item.Subject)
```

Comment:
That's fair; we just need a way to say you invite john@joe.com who is now jackadder in your org.